src/language-js/print/array.js

```diff
@@ -125,4 +136,32 @@ function printArrayItems(path, options, printPath, print) {
   return printedElements;
 }
 
+function printArrayItemsUsingFill(path, options, printPath, print) {
```

Comment:
add tests for this method?

Reply:
We usually don't unit-test individual functions and rely on snapshot testing instead. See the tests I added to this PR.